.circleci/config.yml

```diff
@@ -16,12 +16,12 @@ jobs:
           name: 'Pylint'
           command: |
             source ~/.virtualenvs/tap-harvest/bin/activate
-            pylint tap_harvest --disable missing-docstring,logging-format-interpolation,no-member,broad-except,redefined-variable-type,too-many-branches,too-few-public-methods,wrong-import-order,too-many-locals,consider-using-f-string
+            pylint tap_harvest --disable 'missing-docstring,logging-format-interpolation,no-member,broad-except,redefined-variable-type,too-many-branches,too-few-public-methods,wrong-import-order,too-many-locals,consider-using-f-string,line-too-long,too-many-arguments'
```

Comment:
We added `line-too-long` in disable list. Can we resolve the `line-too-long` warning?

Reply:
`line-too-long` pylint allows a line of 100 chars which reduces the readability of code. So, we followed the autopep-8 standard which allow 120 chars of line.